hibike/devices/ExampleDevice/example_device.cpp

```diff
@@ -3,12 +3,23 @@
 #include "Arduino.h" 
 
 // each device is responsible for keeping track of it's own params
-uint8_t value0;
-uint8_t value1;
-uint16_t value2;
-uint32_t value3;
-uint64_t value4;
-uint64_t value5;
+// This must match the table in hibikeDevices.json
+bool kumiko;
+uint8_t hazuki;
+int8_t sapphire;
+uint16_t reina;
+int16_t asuka;
+uint32_t haruka;
+int32_t kaori;
+uint64_t natsuki;
+int64_t yuko;
+float  mizore;
+double nozomi;
```

Comment:
if we tell them not to use double, do we want to still include an example of double? 
(which we know won't work?)